.github/workflows/publish.yml

```diff
@@ -196,8 +196,8 @@ jobs:
         if: matrix._.platform == 'x86_64-linux' # GitHub actions architecture
         run: |
           gem install pkg/eppo-server-sdk-*.gem --verbose
-          script="puts EppoClient::Core::Client.new(EppoClient::Config.new('placeholder'))"
-          ruby -reppo_client -e "$script" 2>&1 | grep 'fetching new configuration'
+          script="EppoClient::init(EppoClient::Config.new('placeholder')); EppoClient::Client.instance.shutdown"
+          ruby -reppo_client -e "$script" 2>&1 || { echo "❌ Failed to run smoke test"; exit 1 }
```

Comment:
Also added an echo as currently the CI logs don't explain the failure reason and are kind of cryptic:
<img width="1045" alt="image" src="https://github.com/user-attachments/assets/218ee723-db1d-41e6-97f7-9628a09d860d" />